MerlinAU.sh

```diff
@@ -4787,11 +4787,8 @@ _GetNodeInfo_()
 
     if [ $? -ne 0 ]
     then
-        _UpdateLoginPswdCheckHelper_ FAILURE
-        printf "\n${REDct}Login failed for AiMesh Node [$NodeIP_Address].${NOct}\n"
```

Comment:
I think at least some kind of **warning** may be useful here just in case it *IS* an AiMesh node that's having a problem with the login attempt (for some reason).